Project.toml

```
@@ -44,7 +44,7 @@ Conda = "1"
 Contour = "0.5 - 0.6"
 FFMPEG = "0.2 - 0.4"
 FixedPointNumbers = "0.6 - 0.8"
-GR = "0.69.5"
+GR = "0.70"
```

```suggestion
GR = "0.69.5 - 0.70"
```

As @BeastyBlacksmith suggested.

---

Should be ok.